template/tools/get-contributors.sh

```diff
@@ -7,12 +7,14 @@
 # - Removes any usernames with the word "bot" in them.
 # - Removes the trailing comma from the list.
 repo_spec=${1}
-echo "These are the people who have contributed by submitting changes through pull requests :tada:\n\n" > _contributors.qmd
-gh api \
+contributors=$(gh api \
   -H "Accept: application/vnd.github+json" \
   -H "X-GitHub-Api-Version: 2022-11-28" \
   /repos/$repo_spec/contributors \
   --template '{{range .}} [\@{{.login}}]({{.html_url}}){{"\n"}}{{end}}' | \
   grep -v "\[bot\]" | \
   tr '\n' ', ' | \
-  sed -e 's/,$//' >> _contributors.qmd
+  sed -e 's/,$/\n/'
+)
+
+echo "These are the people who contributed by submitting pull requests :tada:\n\n${contributors}"
```

Comment:
Uh, maybe it could even be:
```suggestion
echo "The following people have contributed to this project by submitting pull requests:tada:\n\n${contributors}"
```